dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -2749,6 +2749,8 @@ public String getServerName() {
         return serverToUse;
     }
 
+    //
```

Comment:
these stray comment lines can be removed

Reply:
Removed stray comment.